spec/Scheduler-spec.ts

```diff
@@ -23,16 +23,16 @@ describe('Scheduler.queue', () => {
     let called = false;
     Scheduler.queue.schedule(() => {
       called = true;
-    }, 50);
+    }, 60);
 
     setTimeout(() => {
       expect(called).to.be.false;
-    }, 40);
+    }, 20);
 
     setTimeout(() => {
       expect(called).to.be.true;
       done();
-    }, 70);
+    }, 100);
```

Comment:
Just give small additional buffer between intervals, mobile browser tests are run under emulator, seems timing can be inaccurate (from 50, 40 to 60, 20, give sufficient buffer)